content/concepts/nat.md

```diff
@@ -53,6 +53,27 @@ While the [identify protocol][spec_identify] described above lets peers inform e
 Once again, other peers can help us observe our situation, this time by attempting to dial us at our observed addresses. If this succeeds, we can rely on other peers being able to dial us as well and we can start advertising our listen address.
 
 A libp2p protocol called AutoNAT lets peers request dial-backs from peers providing the AutoNAT service.
+The basic idea as per the implementation in `go-libp2p` is as follows:
+
+- An AutoNAT client send it’s list of [dialable addresses][address_link] to an AutoNAT server and asks the AutoNAT server to
+  see if it can dial the client back on one of those addresses.
```

Comment:
```suggestion
- An AutoNAT client sends its list of [dialable addresses][address_link] to an AutoNAT server and asks the AutoNAT server to see if it can dial the client back on one of those addresses.
```